### DISCUSSION TITLE
Update email alert design based on the new frontend design

### DISCUSSION THREAD

**Discussion by siccovansas** [openstate/jodal#192] [Ideas]
The email alert should present a document in the same way as a document 'card' in the new frontend design.

The subject of the email alert should also use the newly available 'name' of an alert instead of only the query.

**Comment by coret** [comment]
- Maybe use the service name in sender, instead of `Open State Foundation <developer@openstate.eu>`

- The "Alert voor ** stopzetten" text is a poorly formatted link and very close to the document link. Especially on mobile a mis tap on either could happen easily. Maybe make it a bigger red CTA button, with enough space from other links.

- When I received an alert today, I had an urge to add another monitor. But there is no CTA link/button to add a monitor. Clicking the document shows the Bron page for a short time but redirects to the source document. The URL of the service https://bron.live/  isn't part of the email.

![image](https://github.com/user-attachments/assets/ad53dcb1-3720-426b-97c6-0ca067f429d7)

**Comment by Coretteket** [comment]
+1 for @coret's notes on the unsubscribe links. We should probably require a confirmation on the unsubscribe page, or at least send a confirmation mail about the unsubscribing so that it doesn't go unnoticed.
